flycheck.el

```diff
@@ -7182,7 +7182,7 @@ See Info Node `(elisp)Byte Compilation'."
 
 Variables are taken from `flycheck-emacs-lisp-checkdoc-variables'."
   `(progn
-     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(symbol-value opt)))
+     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(custom-quote (symbol-value opt))))
```

Comment:
`quote` doesn't eval it's argument. Also we don't want to quote keywords/strings and so on...